style.css

```diff
@@ -0,0 +1,140 @@
+@import url('https://fonts.googleapis.com/css2?family=Gideon+Roman&display=swap');
+
+:root {
+    --primary: #F8F9FA;
+    --theme: #212529;
+    --light-grey: #CED4DA;
+    --little-dark-grey: #495057;
+    --little-light-grey: #ADB5BD;
+    --bottom-shadow: 0px 15px 10px -15px var(--light-grey);
+    --right-bottom-shadow: 15px -10px 15px -15px var(--little-dark-grey);
+    --all-side-shadow: 0px 0px 2px 2px var(--little-light-grey);
+    --all-side-bold-shadow: 0px 0px 10px 10px var(--little-dark-grey);
+    --font: 'Gideon Roman', cursive;
+}
+
+body {
+    background-color: var(--theme);
+    color: var(--primary);
+}
+
+.navbar {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    gap: 1rem;
+    box-shadow: var(--bottom-shadow);
+    font-family: var(--font);
+}
+
+.nav-brand {
+    padding-left: 1rem;
+}
+
+a {
+    text-decoration: none;
+    color: var(--primary);
+}
+
+.nav-collapse {
+    padding: 1rem;
+}
+
+.nav-list-group {
+    list-style: none;
+    display: inline-flex;
+}
+
+.nav-collapse .list-ele {
+    padding-left: 1rem;
+}
+
+.main {
+    padding: 1rem;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: auto;
+    font-family: var(--font);
+}
+
+.main-container {
+    padding: 1rem;
+    display: flex;
+    flex-wrap: wrap;
+    gap: 2rem;
+}
+
+/* .quiz-type{
+    
+} */
+.quiz-type-one {
+    padding: 1rem;
+    width: 20rem;
+    height: 15rem;
+    background-image: url(/assets/general.jpg);
+    background-repeat: no-repeat;
+    background-attachment: scroll;
+    background-position: center center;
+    background-size: cover;
+    border-radius: 1rem;
+    color: var(--primary);
+}
+
+.quiz-type-one,
+.quiz-type-two,
+.quiz-type-three {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+
```

Comment:
why are you repeating the same properties for question-type-one and question-type-two? combine both classes that have the same property.

Reply:
Sure I will Do it